daemons/controld/controld_execd.c

```diff
@@ -1816,6 +1832,24 @@ do_lrm_invoke(long long action,
             do_lrm_delete(input, lrm_state, rsc, from_sys, from_host,
                           crm_rsc_delete, user_name);
 
+        } else if (pcmk__str_any_of(operation, CRMD_ACTION_RELOAD,
+                                    CRMD_ACTION_RELOAD_AGENT, NULL)) {
+            /* Pre-2.1.0 DCs will schedule reload actions only, and 2.1.0+ DCs
+             * will schedule reload-agent actions only. In either case, we need
+             * to map that to whatever the resource agent actually supports.
+             * Default to the OCF 1.1 name.
+             */
+            struct ra_metadata_s *md = NULL;
+            const char *reload_name = CRMD_ACTION_RELOAD_AGENT;
+
+            md = controld_get_rsc_metadata(lrm_state, rsc, true);
+            if ((md != NULL)
+                && !pcmk_is_set(md->ra_flags, ra_supports_ocf_1_1)
+                && pcmk_is_set(md->ra_flags, ra_supports_reload)) {
```

Comment:
maybe add
&& !pcmk_is_set(md->ra_flags, ra_supports_reload_agent)

Reply:
I don't think I want to support reload-agent in OCF 1.0 agents, since it didn't exist even as an extension before OCF 1.1 ... that would most likely be someone forgetting to update their version tag when updating their agent, and I'd rather them figure that out

Reply:
was just for consistency but I'm fine with strict